PostGIS/Dockerfile-beta.template

```
@@ -0,0 +1,58 @@
+# vim:set ft=dockerfile:
+#
+# Copyright The CloudNativePG Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM postgis/postgis:%%POSTGIS_IMAGE_VERSION%%
+
+# Do not split the description, otherwise we will see a blank space in the labels
+LABEL name="PostgreSQL + PostGIS Container Images" \
+      vendor="The CloudNativePG Contributors" \
+      version="${PG_VERSION}" \
+      release="53" \
+      summary="PostgreSQL + PostGIS Container images." \
+      description="This Docker image contains PostgreSQL, PostGIS and Barman Cloud based on PostGIS %%POSTGIS_IMAGE_VERSION%%."
+
+LABEL org.opencontainers.image.description="This Docker image contains PostgreSQL and Barman Cloud based on PostGIS %%POSTGRES_IMAGE_VERSION%%."
```

```suggestion
LABEL org.opencontainers.image.description="This Docker image contains PostgreSQL and Barman Cloud based on PostGIS %%POSTGIS_IMAGE_VERSION%%."
```
@sxd, POSTGRES_IMAGE_VERSION is not set in this repository